### PR TITLE
style: Replace Ant icons with Lemon ones pt. 2

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

Same premise as https://github.com/PostHog/posthog/pull/13738.
Legacy components removed on the way: `PayCard`, `NPSPrompt`, `FeedbackCallCTA`.
New Lemon UI feature: `LemonTable` now supports a new column attribute by the name of `tooltip`.

### CODE REVIEW

**Review by thmsobrmlr — APPROVED** [review]
Awesome!

Hope you don't mind me adding a commit that adds the tooltip to storybook examples - I've been using that to test.

**Review by pauldambra — APPROVED** [review]
:shipit:

File: frontend/src/scenes/persons/Person.tsx [hunk 1]
```diff
@@ -253,13 +254,10 @@ export function Person(): JSX.Element | null {
...
                         id={person.id}
                         caption={
-                            <div>
-                                <InfoCircleOutlined style={{ marginRight: '.25rem' }} />
-                                <span>
-                                    This page only shows changes made by users in the PostHog site. Automatic changes
-                                    from the API aren't shown here.
-                                </span>
-                            </div>
+                            <AlertMessage type="info">
```
[the remark this answers is not in the document]
Comment (Twixes) [reply frontend/src/scenes/persons/Person.tsx:257]: Oh yeah, I was thinking `LemonNotice`… maybe?

File: frontend/src/lib/components/CloseButton.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { CloseOutlined } from '@ant-design/icons'
 
+// TODO: Remove, but de-ant PropertyFilterButton and SelectGradientOverflow first
 export function CloseButton(props: Record<string, any>): JSX.Element {
     return (
         <span {...props} className={'btn-close cursor-pointer ' + (props.className ?? '')} style={{ ...props.style }}>
```
[the remark this answers is not in the document]
Comment (Twixes) [reply frontend/src/lib/components/CloseButton.tsx:6]: Given it's slated for removal, I'm okay with the warning – I just want to nuke this 😄

**Pull request merged**